import_activity_events.py

```diff
@@ -59,7 +65,7 @@
 """
 
 Q_CREATE_EVENTS_TABLE = """
-    CREATE TABLE IF NOT EXISTS activity_events (
+    CREATE TABLE IF NOT EXISTS activity_events_{sample_rate} (
```

Comment:
Hmm, good point.  TBH I think many of the existing queries may need to move into rollups anyway.  Let's with @davismtl about this before making any final decisions (and feel free to do so without me if the timing works out, since I'm PTO next week)